src/main/java/SkytilsInstallerFrame.java

```diff
@@ -84,7 +84,7 @@ public SkytilsInstallerFrame() {
             testPanel.add(chooser, BorderLayout.EAST);
 
             // Footer
-            footer = new FooterPanel(w / 2, h / 11, 16, 16, new Color(0x8193ff));
+            footer = new FooterPanel(w / 2 + (getOperatingSystem( ) == OperatingSystem.LINUX ?  60  : 0) , h / 11, 16, 16, new Color(0x8193ff));
```

Comment:
This seems like a hack that shouldn't be necessary, are you sure this is the case for the majority of linux users/fonts?

Reply:
its a hack-ish i dont think its possible to change the size based on font width and this way its almost guaranteed that everything is visible unlike before where you can't even see half of the cancel button